src/App/ExtensionContainer.h

```diff
@@ -39,7 +39,7 @@ class Extension;
  * impossible to handle multiple inheritance in the C-API for python extensions. Also using multiple
  * parent classes in python is currently not possible with the default object approach.
  *
- * The concept of extensions allow to circumvent those problems. Extensions are FreeCAD objects
+ * The concept of extensions allow one to circumvent those problems. Extensions are FreeCAD objects
```

Comment:
Subtle grammar: "concept" is singular even though "extensions" is plural, and it's the "concept" we're talking about here.
```suggestion
 * The concept of extensions allows one to circumvent those problems. Extensions are FreeCAD objects
```